src/librustc/hir/intravisit.rs

```diff
@@ -123,9 +122,8 @@ impl<'this, 'tcx> NestedVisitorMap<'this, 'tcx> {
     /// e.g., item, impl-item.
     pub fn inter(self) -> Option<&'this Map<'tcx>> {
         match self {
-            NestedVisitorMap::None => None,
-            NestedVisitorMap::OnlyBodies(_) => None,
             NestedVisitorMap::All(map) => Some(map),
+            _ => None
```

Comment:
Usually we try to make match statements exhaustive in order to catch errors when new variants are added.